build/syntax.go

```diff
@@ -472,21 +461,20 @@ func (x *LoadStmt) Span() (start, end Position) {
 	return x.Load, x.Rparen
 }
 
-// A FuncDef represents a function definition expression: def foo(List):.
-type FuncDef struct {
+// A DefStmt represents a function definition expression: def foo(List):.
+type DefStmt struct {
```

Comment:
This doesn't match with https://github.com/google/skylark/blob/master/syntax/syntax.go#L140

Reply:
Done.